_posts/<spaceranger-23>/2020-09-04-first-reflection-<spaceranger-23>.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: post
+author: spaceranger-23
+title:  "Spaceranger-23 First Reflection!"
+---
+
+# The first week assignments really challanged my brain on how I think about solving problems. After taking previous python courses online, I assumed that the first assignment would start with print("Hello, World"). I think it was awesome to write commands to direct turtles to execuate a movement. I did struggle at first with understanding how to input coordinates to direct a turtle to a certain area, but after a while the system started to make since to me. There is one thing I know going foward and that is I can think that I will create a complete program in minutes or hours. There is going to be a lot of trail and error that I will have to make myself work through. Great first assignment and hopefully I am able to execute the first pull request on Github. 
```

Comment:
Your `#` makes everything that follows get formatted like a title.

# In other words, like this.

No worries, but this is something you can fix in the future when we go to revise and clean up posts.